apps/emqx/src/emqx_cm.erl

```diff
@@ -592,6 +593,26 @@ channel_with_session_table(ConnModuleList) ->
         sets:is_element(ConnModule, ConnModules)
     ]).
 
+%% @doc Get clientinfo for all clients, regardless if they use clean start or not.
+all_channels_table(ConnModuleList) ->
```

Comment:
why this name?

Reply:
Because it enumerates all channels, regardless of `clean_start`.

This is different from `channel_with_session_table`.